code/modules/crafting/recipes.dm

```diff
@@ -133,6 +133,18 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 
+/datum/crafting_recipe/minigun
+	name = "Laser Minigun"
+	result = /obj/item/minigunpack2
+	reqs = list(/obj/item/gun/energy/laser = 3,
```

Comment:
Make these carbines so that they cant be made with things like practice or such. Also gates possible spam of making it via book warden has might be nice.